### PR TITLE
making the functional-account configurable

### PR DESCRIPTION
**Pull request by valerioj**
the "functional account" was hardcoded and now it's a value in config.yaml, with the key "openshift_functional_account"

### CODE REVIEW

File: config.yaml.example [hunk 1]
```diff
@@ -27,6 +27,7 @@ newrelic_api_token:
 newrelic_api_account:
 openshift_chargeback_sender:
 openshift_chargeback_art:
+openshift_functional_account:
```
Comment (ganto) [inline config.yaml.example:30]: I'm not really happy with this name. Maybe something like `openshift_additional_project_admin_account` is better? 

And then there must be a clear description what this means. One month from now or someone not involved in this change won't understand at all what's that setting about. We need to get better in that generally...

File: server/openshift/project.go [hunk 2]
```diff
@@ -286,7 +288,8 @@ func validateProjectPermissions(clusterId, username, project string) error {
 	}
 
 	// Allow functional account
-	if username == "fssnow1" {
+	functionalAccount := config.Config().GetString("openshift_functional_account")
+	if username == functionalAccount {
```
Comment (ganto) [inline server/openshift/project.go:292]: I'm not fully sure about the behaviour of the code here. What if this configuration option is missing in the config? What if I set an empty string? Is it possible for keycloak to return an empty string for the username somehow? Is there no way of unintended behaviour possible? Maybe we still could add some safety checks?

Comment (valerioj) [reply server/openshift/project.go:292]: i'm creating some tests on the go... i will make further commits

File: config.yaml.example (hunk 1 above)
Comment (valerioj) [reply config.yaml.example:30]: i will change the name to what you have suggested

Comment (valerioj) [reply config.yaml.example:30]: where do you suggest that the description of this variable is? in config.yaml.example? in project.go? in any documentation?

Comment (ganto) [reply config.yaml.example]: YAML allows comments, so I guess there would be a good place.

File: server/openshift/project.go (hunk 2 above)
Comment (ganto) [reply server/openshift/project.go]: Looks much better 👍

File: config.yaml.example (hunk 1 above)
Comment (ganto) [reply config.yaml.example]: You still want to add the documentation and example file rename? After that I think we're good to go.

Comment (valerioj) [reply config.yaml.example]: Yeah I can do that on Monday and we can try to merge, we will be good to... "Go"

Comment (valerioj) [reply config.yaml.example]: i just renamed the file and added just a couple of basic comments. with my level of knowledge about this project so far, that's what i could do. feel free to review/approve

**Comment by ganto** [comment]
Ready for merging. After that we can make a new release.

**Pull request merged**